pkg/cmd/variables/variables.go

```diff
@@ -103,6 +103,7 @@ func NewCmdVariables() (*cobra.Command, *Options) {
 	}
 	o.DiscoverFromGit = true
 	cmd.Flags().StringVarP(&o.File, "file", "f", filepath.Join(".jx", "variables.sh"), "the default variables file to lazily create or enrich")
+	cmd.Flags().StringVarP(&o.Repository, "app", "", "", "Name of the app or repository")
```

Comment:
Remember to run `make docs` to generate docs.

Reply:
done